### DISCUSSION TITLE
Saving view configuration and figures

### DISCUSSION THREAD

**Discussion by lee-t** [vitessce/vitessce-python#200] [Ideas]
We have a case where a user might make edits to a view config in a widget generated by pyvitessce ie resizing the windows. Is it then possible to then print out the modified config similar to how Vitessce.io editor does? It would be useful from a replication standpoint that someone using the data viewer would be able to save the current config with all the selections, cell list expressions and views for later retreival.
On a similar note, aside from system screenshots, there seems to be no means of saving the views as plots or figures. I dont know if there are any plans in the design of vitessce to be able to save views as figures.

**Comment by keller-mark** [comment]
You can access the modified config (in its JSON / python dict format) this way:

```py
vc = VitessceConfig(name="My config")
# initial configuration here...
vw = vc.widget()
# user interactions here...
modified_config = vw.config
```

**Comment by keller-mark** [comment]
As for the saving figures to image files, we have the following issues for the feature request https://github.com/vitessce/vitessce/issues/975 https://github.com/vitessce/vitessce/issues/1288 but there is currently not support.
